components/wb-jsonmanager/index.json-ld

```diff
@@ -22,12 +22,12 @@
 			{
 				"title": "JSON Manager",
 				"language": "en",
-				"path": "jsonmanager-en.html"
+				"path": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/wb-jsonmanager/jsonmanager-en.html"
```

Comment:
Since the links are wrongly generated by the script in both index pages (en/fr) the parameter "path" must be replaced by "url" from line 25 to 52. The "path" parameter is used for internal links inside GCWeb and "url" for external links.

Reply:
fixed